templates/joomla/html/com_content/article/requirements.php

```diff
@@ -133,6 +133,80 @@
 		<div class="alert alert-info"><?php echo JText::sprintf('TECHNICAL_REQUIREMENTS_SUPPORTED_BROWSERS', JHtml::_('link', 'https://docs.joomla.org/S:MyLanguage/Joomla_Browser_Support', 'https://docs.joomla.org/Joomla_Browser_Support', ['class' => 'alert-link'])); ?></div>
 
 		<h2><?php echo JText::_('TECHNICAL_REQUIREMENTS_HEADING_REQUIREMENTS_FOR_UNSUPPORTED_SOFTWARE'); ?></h2>
+        <h3><?php echo JText::_('TECHNICAL_REQUIREMENTS_HEADING_REQUIREMENTS_FOR_JOOMLA_4'); ?></h3>
+        <div class="alert alert-warning"><?php echo JText::_('TECHNICAL_REQUIREMENTS_ALERT_UNRELEASED_VERSION'); ?></div>
+        <table class="table table-bordered table-striped">
+            <thead>
+            <tr>
+                <th scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_SOFTWARE'); ?></th>
+                <th scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_RECOMMENDED'); ?></th>
+                <th scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_MINIMUM'); ?></th>
+                <th class="hidden-phone" scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_MORE_INFORMATION'); ?></th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <th scope="row">PHP</th>
+                <td>7.4</td>
+                <td>7.2</td>
+                <td class="hidden-phone">
+                    <a href="https://www.php.net" target="_blank" rel="noopener noreferrer">https://www.php.net</a>
+                </td>
+            </tr>
+            </tbody>
+            <tbody>
+            <tr>
+                <th scope="rowgroup" colspan="4"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_SUPPORTED_DATABASES'); ?></th>
+            </tr>
+            <tr>
+                <th scope="row">MySQL</th>
+                <td>5.6 +</td>
```

Comment:
```suggestion
                <td>5.7 +</td>
```
We shouldn't suggest EOL versions

Reply:
This was the version approved by Production